.coveragerc

```diff
@@ -12,10 +12,4 @@ show_missing = true
 
 [run]
 branch = true
-cover_pylib = false
 plugins = Cython.Coverage
-relative_files = true
```

Comment:
Hm.. Perhaps this relative files setting could be a workaround for codecov not understanding paths… Could be an adequate repro for a bug report.